python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -78,6 +78,16 @@ class Value extends TObject {
     predicate isBuiltin() {
         this.(ObjectInternal).isBuiltin()
     }
+    
+        /** INTERNAL -- Do not use */
```

Comment:
Is this still needed? (It doesn't appear to be the case for the present PR, but perhaps there are other PRs that could use this functionality?)
If not, I would suggest just removing it.